DaS-PC-MPChan/IRCClient.vb

```diff
@@ -53,21 +53,48 @@ Public Class IRCClient
         blackSet.Add(self.SteamId)
 
         Dim candidates As New List(Of DSNode)
+
+        Dim DkmSelectedPref As DkmPref = DSCM.DkmPrefBox.SelectedItem
+
         For Each t In ircNodes.Values
             Dim node As DSNode = t.Item1
+
+            'WIP: Darkmoon proprocessing for Blacklist
+            If DkmSelectedPref.Value <> 0 Then
+                If DSCM.DkmCheckBoxSinnersOnly.Checked Then
+                    If node.Indictments < 1 Then
+                        blackSet.Add(node.SteamId)
```

Comment:
I figured that sooner or later there would end up to be a lot of "If ... Continue For" and a copy/paste of adding to the blackSet seemed more uniform and could aggregate all of the skips. Now that I think of it, there would be an add and lookup cost for the hash set.